tests/Aspire.Playground.Tests/ProjectSpecificTests.cs

```diff
@@ -68,26 +67,27 @@ public async Task AzureFunctionsTest()
         await app.StartAsync();
         await app.WaitForResources().WaitAsync(TimeSpan.FromMinutes(2));
 
-        // Wait for the blobTrigger to be discovered as an indication that the host and worker
-        // has been successfully launched
+        // Wait for the 'Job host started' message as an indication
+        // that the Functions host has initialized correctly
         await WaitForAllTextAsync(app,
             [
-                "MyAzureBlobTrigger: blobTrigger"
+                "Job host started"
```

Comment:
Should we wait for both? Though I think this combined with the http resilience will be enough.

Reply:
IMO, checking for `"MyAzureBlobTrigger: blobTrigger"` doesn't have as much value as running the actual blob trigger scenario so that's why I opted to change the check that we do here.